picard/ui/scripteditor.py

```diff
@@ -288,7 +290,7 @@ def __init__(self, parent=None, examples=None):
 
         self.FILE_TYPE_ALL = _("All Files") + " (*)"
         self.FILE_TYPE_SCRIPT = _("Picard Script Files") + " (*.pts *.txt)"
-        self.FILE_TYPE_PACKAGE = _("Picard Naming Script Package") + " (*.pnsp *.json)"
+        self.FILE_TYPE_PACKAGE = _("Picard Naming Script Package") + " (*.pnsp *.yaml)"
```

Comment:
This may be a dumb question, but is there a possibility in the future of also saving tagging scripts in this same format?  If so, should the description be changed to "Picard Script Package" and the extension to (perhaps) "*.psp"?

Reply:
Yes, I think definitely, that should be a goal. What about `*.ptsp` ? So we have `.pts` for Picard Tagger Script, and `.ptsp`  for Picard Tagger Script Package?